main.go

```diff
@@ -32,16 +34,15 @@ func main() {
 	if err != nil {
 		errAndExit("Improperly formatted JSON file: %v", err)
 	}
-	fmt.Println("Please input your master password")
-	var reader = bufio.NewReader(os.Stdin)
-	password, err := reader.ReadBytes('\n')
+	fmt.Print("Please input your master password: ")
+	password, err := term.ReadPassword(int(syscall.Stdin))
+	fmt.Println()
```

Comment:
A really minor thing, but could you remove this `Println` and make the above `Print` into `Println` instead?